public/services/FindingsService.ts

```diff
@@ -16,9 +16,9 @@ export default class FindingsService {
   }
 
   getFindings = async (
-    detectorParams: GetFindingsParams
+    detectorParams?: GetFindingsParams
   ): Promise<ServerResponse<GetFindingsResponse>> => {
-    const { detectorType, detectorId } = detectorParams;
+    const { detectorType, detectorId } = detectorParams || {};
```

Comment:
Nitpick: Rather than declaring `detectorParams` as a nullable parameter, I believe you could just refactor line 19 to `detectorParams: GetFindingsParams = {}` to provide a default value when the parameter is null. This would then eliminate the need for the `|| {}` at the end of line 21.